rosdep/python.yaml

```diff
@@ -101,6 +101,10 @@ azure-storage-file-share-pip:
   ubuntu:
     pip:
       packages: [azure-storage-file-share]
+bluerobotics-ping-pip:
```

Comment:
Please make the name of this key `python3-bluerobotics-ping-pip`

https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1

Reply:
The entry needs to be re-alphabetized after the name change.